systemtests/scripts/functions

```diff
@@ -301,6 +301,19 @@ check_files_written()
     fi
 }
 
+check_sd_files_written()
+{
+    LOG=$1
+    FILES="$(awk '/SD Files Written:/ { last=$4 } END { print last }' "$LOG")"
+
+    if [[ "$FILES" -lt 1 || -z "$FILES" ]]; then
+        print_debug "ERROR: Expected SD files written > 0, got $FILES."
```

Comment:
```suggestion
        print_debug "ERROR: Expected number of SD files written > 0, got \"$FILES\"."
```

Reply:
In case $FILES is empty this makes more sense.

Reply:
Good suggestions, made the change. Thanks for the review.